* Use AC_REQUIRE instead of AC_BEFORE (bug #297943).
* Do not introduce compiler optimisation flags.


--- a/aclocal.m4
+++ b/aclocal.m4
@@ -93,7 +93,7 @@
 dnl
 AC_DEFUN(AC_LBL_C_INIT,
     [AC_PREREQ(2.12)
-    AC_BEFORE([$0], [AC_PROG_CC])
+    AC_REQUIRE([AC_PROG_CC])
     AC_BEFORE([$0], [AC_LBL_FIXINCLUDES])
     AC_BEFORE([$0], [AC_LBL_DEVEL])
     AC_BEFORE([$0], [AC_QOSIENT_DEBUG])
@@ -126,7 +126,7 @@
     if test "$GCC" = yes ; then
        if test "$SHLICC2" = yes ; then
           ac_cv_lbl_gcc_vers=2
-          $1="-O3"
+          $1=""
        else
           AC_MSG_CHECKING(gcc version)
           AC_CACHE_VAL(ac_cv_lbl_gcc_vers,
@@ -137,7 +137,7 @@
             -e 's/\..*//'`)
           AC_MSG_RESULT($ac_cv_lbl_gcc_vers)
           if test $ac_cv_lbl_gcc_vers -gt 1 ; then
-             $1="-O3"
+             $1=""
           fi
        fi
     else
